torch/lib/c10d/Utils.cpp

```diff
@@ -344,16 +352,12 @@ std::tuple<int, std::string> accept(
   int socket;
   SYSCHECK_ERR_RETURN_NEG1(socket = ::accept(listenSocket, NULL, NULL))
 
-  // Get address of the connecting process
-  struct ::sockaddr_storage addr;
-  socklen_t addrLen = sizeof(addr);
-  SYSCHECK_ERR_RETURN_NEG1(::getpeername(
-      socket, reinterpret_cast<struct ::sockaddr*>(&addr), &addrLen))
-
   setSocketNoDelay(socket);
 
-  return std::make_tuple(
-      socket, sockaddrToString(reinterpret_cast<struct ::sockaddr*>(&addr)));
+  // Get address of the connecting process
+  std::string socketAddr = getSocketAddr(socket);
```

Comment:
Just call `getSocketAddr(socket)` in make_tuple to avoid the extra variable.

Reply:
ok